sysutils/mc/Portfile

```diff
@@ -37,10 +37,6 @@ patchfiles          patch-src_subshell_common.c.diff
 
 configure.args      --without-x --enable-vfs-sftp=no
 
-if {[vercmp ${macos_version} 10.13] >= 0} {
```

Comment:
If we're removing this workaround, I think we will still need to increment `revision` for affected systems in order rebuild without and update the port archive:

```
if {[vercmp ${macos_version} 10.13] >= 0} {
    revision [expr ${revision}+1]
}

```

@jmroot can confirm if this is needed.